language/stdlib/modules/DiemTransactionPublishingOption.move

```diff
@@ -139,6 +127,44 @@ module DiemTransactionPublishingOption {
         include DiemConfig::SetAbortsIf<DiemTransactionPublishingOption>{account: dr_account};
     }
 
+    /// If published, halts transactions from all accounts except DiemRoot
+    resource struct HaltAllTransactions {}
+
```

Comment:
Nit: could this be declared along with the other resources at the top of this module?